tutorials/3-run-node/index.md

```diff
@@ -314,8 +314,10 @@ Despite this initial balance, before you run your blockchain you still need to e
 
 <HighlightBox type="note">
 
-In this scenario you must meet the 2/3 threshold for validation, so you must stake at least `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all your stake, so you still have tokens to pay for gas.
-<br></br>
+In this scenario, for your network to even run, you must meet the 2/3 threshold of the weighted validators.
+<br/><br/>
+However you will be alone on the network, so you can stake any number. However, to remind yourself that it is important honest nodes stake a large amount, you stake `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all your stake, so you still have tokens to pay for gas.
```

Comment:
However you will be alone on the network, so you can stake any number. However, to remind yourself that it is important that honest nodes stake a large amount, you stake `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all of your tokens so you can still pay for gas and so you don't run out of tokens later.